### PR TITLE
Input files are positional arguments.

### PR DESCRIPTION
**Pull request by snim2**
Please check before merge...

### CODE REVIEW

**Comment by vext01** [comment]
Looks good. I'll test this after I have kicked off the experiment.

File: mark_outliers_in_json.py [hunk 1]
```diff
@@ -6,15 +6,16 @@
 usage: Write lists of outliers into Krun results file(s).
 Example usage:
 
-$ python mark_outliers_in_json.py -f results1.json.bz2
-$ python mark_outliers_in_json.py -f results1.json.bz2 -f results2.json.bz2 --window 250
-       [-h] [--filename JSON_FILES] [--window WINDOW_SIZE]
+$ python mark_outliers_in_json.py results1.json.bz2
+$ python mark_outliers_in_json.py ---window 250 results1.json.bz2 results2.json.bz2
+        [-h] [--window WINDOW_SIZE] [--threshold THRESHOLD] json_files
```
Comment (vext01) [inline mark_outliers_in_json.py:11]: ah, can we use the notation: `[json_file_1] [...] [json_file_n]` in the usage hints?

Comment (snim2) [inline mark_outliers_in_json.py:11]: Help text is automatically generated by the `argparse` module, this file-level documentation is copied from that

Comment (vext01) [inline mark_outliers_in_json.py:11]: Ah fair enough.

**Comment by vext01** [comment]
In both scripts:
- In `create_cli_parser()` you need to update the example usage.
- Works with one positional arg but not two.

**Comment by snim2** [comment]
@vext01 er, I'm not sure why git put commit f66eacb on this branch. Will sort it out during squash

**Comment by vext01** [comment]
So does this now accept >1 positional arg?

**Comment by snim2** [comment]
Sorry, I misunderstood your last comment. Watch this space.

**Comment by vext01** [comment]
OK, go ahead and do the ol' squisharoo.

**Comment by snim2** [comment]
Squished.

**Comment by snim2** [comment]
Thanks.

**Pull request merged**